ArraysHomework/ArraysHomework.cs

```diff
@@ -69,23 +77,26 @@ public void Arrays()
                 
             }
 
+            int[] combinedArray = new int[50];
+            combinedArray = array1.Concat(array2).ToArray();
```

Comment:
This is a perfectly valid solution, and it's an approach that I might take in real code. But what is it actually doing?
How would you solve this using the basic tools of arrays and loops, without using Linq methods (methods that need the `using System.Linq` directive at the top of the file)?